lib/folio_sync/folio/writer.rb

```diff
@@ -10,6 +10,18 @@ def initialize
       def update_instance_record(instance_id, instance_record)
         @client.put("/instance-storage/instances/#{instance_id}", instance_record)
       end
+
+      def check_out_item_by_barcode(barcode, repo_key)
+        folio_requests_config = Rails.configuration.folio_requests[:repos][repo_key.to_sym]
+
+        payload = {
```

Comment:
`loanDate` is not part of the payload as it defaults to the current time automatically